app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewDialog.java

```diff
@@ -153,7 +154,8 @@ public void goToLinkedPage(boolean inNewTab) {
         if (overlayView == null && containerView != null) {
             overlayView = new LinkPreviewOverlayView(getContext());
             overlayView.setCallback(new OverlayViewCallback());
-            overlayView.setPrimaryButtonText(getStringForArticleLanguage(pageTitle, R.string.button_continue_to_article));
+            overlayView.setPrimaryButtonText(getStringForArticleLanguage(pageTitle,
+                    pageTitle.namespace() == Namespace.TALK || pageTitle.namespace() == Namespace.USER_TALK ? R.string.button_continue_to_talk_page : R.string.button_continue_to_article));
```

Comment:
![Screenshot_20200929-123747_Wikipedia Dev](https://user-images.githubusercontent.com/2435576/94607499-d76fce00-0250-11eb-91cc-6b2bab359b1b.jpg)

Not sure but should this also work on the Gallery?

[Barack Obama] -> Gallery -> the credit of the 6th image

Reply:
done! good catch.